Dockerfile.web

```
@@ -0,0 +1,118 @@
+COPY apps/web-server/src apps/web-server/src
+
+# Build server
+RUN cd apps/web-server && npm run build
+
+# Stage 3: Python backend
+FROM python:3.12-slim-bookworm AS python-builder
+
+WORKDIR /app
+
+# Install git and other dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)